### PR TITLE
stats plugin text

### PR DESCRIPTION
**Pull request by brianteeman**
Improve the text of the message

specifically it is wrong to say "this plugin" in the context of where the message is displayedd

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_system_stats.ini [hunk 1]
```diff
@@ -24,7 +24,7 @@ PLG_SYSTEM_STATS_MODE_OPTION_ALWAYS_SEND="Always send"
 PLG_SYSTEM_STATS_MODE_OPTION_NEVER_SEND="Never send"
 PLG_SYSTEM_STATS_MODE_OPTION_ON_DEMAND="On demand"
 PLG_SYSTEM_STATS_MSG_ALLOW_SENDING_DATA="Enable Joomla Statistics?"
-PLG_SYSTEM_STATS_MSG_JOOMLA_WANTS_TO_SEND_DATA="In order to better understand our install base and end user environments, this plugin has been created to send those statistics back to a Joomla! controlled central server. No identifying data is captured at any point. You can change these settings later from Plugins > System - Joomla! Statistics."
+PLG_SYSTEM_STATS_MSG_JOOMLA_WANTS_TO_SEND_DATA="In order to better understand our install base and end user environments it is helpful if you send some site information back to a Joomla! controlled central server. No identifying data is captured at any point. You can change these settings later from Plugins > System - Joomla! Statistics."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_system_stats.ini]: I would put a comma after `environments`, otherwise, it looks good.

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_system_stats.ini]: there is no need for the comma

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f3f7ed41c540f760eec15124c0affab5900f6c5b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15735">issues.joomla.org/tracker/joomla-cms/15735</a>.</sub>

**Comment by tonypartridge** [comment]
I have tested this item :white_check_mark: successfully on f3f7ed41c540f760eec15124c0affab5900f6c5b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15735">issues.joomla.org/tracker/joomla-cms/15735</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**